contracts/reputation/Reputation.sol

```diff
@@ -226,6 +236,24 @@ contract Reputation is ConsensusModule {
         withdrawalCooldownPeriodInBlocks = _withdrawalCooldownPeriodInBlocks;
     }
 
+    function setupConsensus(
```

Comment:
I dont like that there would be a constructor and a setup function together; we can correct this later, but it s a bad pattern.